cmd/zetae2e/local/local.go

```diff
@@ -295,14 +311,14 @@ func localE2ETest(cmd *cobra.Command, _ []string) {
 
 	if testAdmin {
 		eg.Go(adminTestRoutine(conf, deployerRunner, verbose,
-			e2etests.TestWhitelistERC20Name,
-			e2etests.TestPauseZRC20Name,
+			//e2etests.TestWhitelistERC20Name,
+			//e2etests.TestPauseZRC20Name,
 			e2etests.TestUpdateBytecodeZRC20Name,
-			e2etests.TestUpdateBytecodeConnectorName,
-			e2etests.TestDepositEtherLiquidityCapName,
-			e2etests.TestCriticalAdminTransactionsName,
-			e2etests.TestPauseERC20CustodyName,
-			e2etests.TestMigrateERC20CustodyFundsName,
+			//e2etests.TestUpdateBytecodeConnectorName,
+			//e2etests.TestDepositEtherLiquidityCapName,
+			//e2etests.TestCriticalAdminTransactionsName,
+			//e2etests.TestPauseERC20CustodyName,
+			//e2etests.TestMigrateERC20CustodyFundsName,
```

Comment:
Need to revert or fix these before merge

Reply:
Yes correct, this was for local experimentation